deploy/kubernetes/releases/csi-digitalocean-dev.yaml

```diff
@@ -118,6 +118,20 @@ provisioner: dobs.csi.digitalocean.com
 
 ---
 
+# NOTE(arslan): this will probably fail , because the CRD is created via the
+# csi-snapshotter sidecar, that is part of the csi-do-controller statefulset.
+# We need to create this seperately.
```

Comment:
Add to this comment some documentation about how other devs need to proceed to get this working (eg "apply this section separately after XYZ")

Reply:
This comment will be removed in upcoming PR's it's just here to remind myself.